bin/ledger2beancount

```diff
@@ -354,30 +354,6 @@ sub push_metadata($$$) {
     push_line $depth, pp_metadata($key, $value);
 }
 
-# add a comment line. Wrapper around push_line() for comments
-sub push_comment($$) {
```

Comment:
Do you really want to remove `push_comment`? it makes sense as a generic helper. I'd rather keep it, but simplify its logic to just be a `push_line` which additionally adds a heading `;`.

Reply:
Ok, I'll keep it